### ISSUE TITLE
[SonicFrontiers] 

### ISSUE THREAD

**Issue by ZolarZ** [thesupersonic16/HedgeModManager#523]
this is all i have :/
![image](https://user-images.githubusercontent.com/90578031/205226009-e4605d42-2765-4ad0-8fc7-45eb686ed7b0.png)

[HMM_Snapshot_00122022030026.txt](https://github.com/thesupersonic16/HedgeModManager/files/10138494/HMM_Snapshot_00122022030026.txt)

HMM Info:
```
    Version: 7.8-2-caf92e8
    Args: 
    StartDir: E:\SteamLibrary\steamapps\common\SonicFrontiers\
    Process Level: Administrator
    Game: Sonic Frontiers
    GameInstall: Sonic Frontiers
    Update Status: Up To Date
```

Exception:
```
    Type: NullReferenceException
    Message: Object reference not set to an instance of an object.
    Source: HedgeModManager
    Function: Void OpenAboutCodeWindow(HedgeModManager.Code)
    StackTrace: 
       at HedgeModManager.MainWindow.OpenAboutCodeWindow(Code code) in D:\a\HedgeModManager\HedgeModManager\HedgeModManager\UI\MainWindow.xaml.cs:line 1845
       at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
       at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
       at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
       at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
       at System.Windows.Controls.Control.HandleDoubleClick(Object sender, MouseButtonEventArgs e)
       at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
       at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
       at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
       at System.Windows.UIElement.ReRaiseEventAs(DependencyObject sender, RoutedEventArgs args, RoutedEvent newEvent)
       at System.Windows.UIElement.OnMouseDownThunk(Object sender, MouseButtonEventArgs e)
       at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
       at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
       at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
       at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
       at System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
       at System.Windows.Input.InputManager.ProcessStagingArea()
       at System.Windows.Input.InputManager.ProcessInput(InputEventArgs input)
       at System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
       at System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawMouseActions actions, Int32 x, Int32 y, Int32 wheel)
       at System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr hwnd, WindowMessage msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
       at System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
       at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
       at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
       at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
       at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
       at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
       at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
       at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
       at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
       at System.Windows.Application.RunDispatcher(Object ignore)
       at System.Windows.Application.RunInternal(Window window)
       at HedgeModManager.HedgeApp.Main(String[] args) in D:\a\HedgeModManager\HedgeModManager\HedgeModManager\HedgeApp.xaml.cs:line 271
    InnerException: 
```

**Comment by RagdollClash** [comment]
Update your codes and make sure anything labeled as "outdated" isn't enabled. Additionally, make sure your mods are updated.

**Comment by RagdollClash** [comment]
Re-opening as this was unrelated to the game patch and instead the codes tab.

**Comment by thesupersonic16** [comment]
Fixed crash in 71da3f2d4b3abd94bb72b5ffb67abe3e14784e5d.

Please reopen the issue if the same issue is still occurring.
